src/test/shell/bazel/bazel_layering_check_test.sh

```diff
@@ -144,11 +144,6 @@ EOF
 }
 
 function test_bazel_layering_check() {
```

Comment:
Just checking: the related test/feature test_bazel_layering_check_header_only() still doesn't work on MacOS?

Reply:
i didn't want to change it in this PR too since it's unrelated, but I think we can enable it afterwards https://github.com/bazelbuild/bazel/pull/22370

Reply:
sgtm